tests/openvino/test_modeling.py

```diff
@@ -430,7 +440,9 @@ class OVModelForFeatureExtractionIntegrationTest(unittest.TestCase):
     def test_compare_to_transformers(self, model_arch):
         model_id = MODEL_NAMES[model_arch]
         set_seed(SEED)
-        ov_model = OVModelForFeatureExtraction.from_pretrained(model_id, export=True)
+        ov_model = OVModelForFeatureExtraction.from_pretrained(
+            model_id, export=True, ov_config={"CACHE_DIR": "", "INFERENCE_PRECISION_HINT": "f32"}
```

Comment:
I suggest using OV_CONFIG const in utils and loading it via:
ov_model = OVModelForFeatureExtraction.from_pretrained(
            model_id, export=True, ov_config=OV_CONFIG)

Reply:
> I suggest using OV_CONFIG const in utils and loading it via:
ov_model = OVModelForFeatureExtraction.from_pretrained(
model_id, export=True, ov_config=OV_CONFIG)

I like being explicit in tests, especially when we deviate from defaults. But I can add an F32_CONFIG at the top of this file.

> also setting NUN_STREAMS = 1 could be useful

Agreed that this can be useful, but we should run some tests and assuming no issues, set this by default in optimum-intel, not just for the tests.

Reply:
> I like being explicit in tests, especially when we deviate from defaults. But I can add an F32_CONFIG at the top of this file.

Done

Reply:
the same will be needed also in stable diffusion tests

Reply:
> the same will be needed also in stable diffusion tests

Thanks @dtrawins I did that in this PR https://github.com/huggingface/optimum-intel/pull/518